changelog/_go-ver-1121.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:change
+core: Bump Go version to 1.18.5.
```

Comment:
Since 1.12.0 hasn't been released yet, should we be modifiying `changelog/_go-ver-1120.txt` rather than adding a new file for 1.12.1?

Reply:
Yeah, I was not sure what to do with this. But, your suggestion makes sense! I am going to remove this file and update the existing one.

Reply:
@ccapurso is correct - editing the file produces a single line in the generated changelog about the go version for 1.12.0. If there are multiple files the generated changelog gets a new line about each go version we bumped.